lib/board.rb

```diff
@@ -50,8 +57,17 @@ def diagonal_coordinates?(coordinates)
   end
 
   def place(ship, coordinates)
+    @existing_coordinates += coordinates
```

Comment:
decided to update the place method to use within overlap. We should make some tests to make sure everything runs smoothly but it didn't affect any other test. += combined the coordinate array with the existing coordinates array.

Reply:
Looks great! I like the update on the place method!